CommunityToolkit.Authentication.Uwp/WebAccountProviderConfig.cs

```diff
@@ -19,15 +21,33 @@ public struct WebAccountProviderConfig
         /// </summary>
         public WebAccountProviderType WebAccountProviderType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the properties that need to be added when constructing <see cref="Windows.Security.Authentication.Web.Core.WebTokenRequest"/> (for MSA).
+        /// </summary>
+        public IDictionary<string, string> MSATokenRequestProperties { get; set; }
+
+        /// <summary>
+        /// Gets or sets the properties that need to be added when constructing <see cref="Windows.Security.Authentication.Web.Core.WebTokenRequest"/> (for AAD).
+        /// </summary>
+        public IDictionary<string, string> AADTokenRequestProperties { get; set; }
```

Comment:
In new repo we have nullability turned-on, so thinking ahead. We probably just want to initialize these with `new()`? Then we don't need null check in the if statement below, can just loop over empty collection if nothing added.

Reply:
I initialized these two properties in the constructor. Since the current C# version of the project is 9, inline property initialization is not available. Maybe in the future we can use the new syntax when we upgrade the C# version.

Reply:
Ah good to know, we could probably bump it easily enough, we've been setting that for all our new stuff we've been working on to take advantage of the latest syntax. Thanks for calling it out!